docs/tutorials/introduction.ipynb

```diff
@@ -304,16 +320,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "a[4]"
+    "ds['row', 1:2]"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "a[4:5]"
+    "Attributes are stored in a dictionary similar to coordinates.\n",
```

Comment:
At first I was confused by the fact that you start talking about attributes here. Am I right that it is because when you slice with a single index, the coord becomes and attribute?

This is quite an advanced behaviour of Scipp, and may be confusing to beginners?
I would suggest to manually add some attributes that differ to `alice` and `bob`, and then show that you can still perform operations with both?